.circleci/config.yml

```diff
@@ -19,12 +19,24 @@ jobs:
           command: |
             npm test
       - run:
-          name: Run showcase test
+          name: Run showcase test for js users
           command: |
             cp -r typescript/test/protos ./.baseline-test-out
             cd .baseline-test-out 
             npm install
-            cd ../typescript/test/test_application
+            npm pack
+            cp -r ../typescript/test/test_application_js ~/
+            cp -r showcase-0.1.0.tgz ~/test_application_js
```

Comment:
no need to use `-r` to copy just one file

Reply:
`test_application_js` is a folder, getting rid of `-r` may give error here

Reply:
I'm writing about line 29, where you copy just one file. `-r` is not needed there.

Reply:
you are right, removed `-r`